fe/projects/Seller.Portal/src/areas/Orders/components/OrderForm/OrderForm.js

```diff
@@ -82,7 +82,7 @@ function OrderForm(props) {
 
     const getProductSuggestionValue = (suggestion) => {
 
-        return "(" + suggestion.sku + ")" + " " + suggestion.name;
+        return suggestion.name;
```

Comment:
**question:** Why you modify this suggestion?

Reply:
Because from be property name has value like this "Anton (An_01)".

![image](https://github.com/user-attachments/assets/1a882ad5-ee70-4640-ac0b-14054cef4994)

Line 52.

Reply:
we should not modify it, please revert